guacamole-docker/bin/start.sh

```diff
@@ -567,6 +567,38 @@ END
     ln -s /opt/guacamole/duo/guacamole-auth-*.jar   "$GUACAMOLE_EXT"
 }
 
+##
+## Adds properties to guacamole.properties witch configure the Cas
+## authentication service.
+##
+associate_cas() {
+    # Verify required parameters are present
+    if [ -z "$CAS_AUTHORIZATION_ENDPOINT" ] || \
+       [ -z "$CAS_REDIRECT_URI" ]
+    then
+        cat <<END
+FATAL: Missing required environment variables
+-----------------------------------------------------------------------------------
+If using the Cas authentication extension, you must provide each of the
```

Comment:
Should probably be consistent about calling this the "CAS authentication extension."

Reply:
yes, it's a typo error.

Can or should I add a new commit to the branch?

What should the commit description be [GUACAMOLE-1082: Add guacamole-auth-cas to docker Script - Cas 2 CAS typo ]?

Reply:
You should be able to just make then changes and then do `git add` and `git commit --amend`, and then you can force push with a `git push -f`, which will update the pull request automatically.